src/models/state-manager.ts

```diff
@@ -2,8 +2,25 @@ import { Unsubscribe } from '@reduxjs/toolkit';
 import { State } from './state';
 import StateListener from './state-listener';
 
+/**
+ * Manages the information contained in the state for an AnswersHeadless instance.
```

Comment:
I think somewhere in the docs for this class we should indicate that the `state` is immutable and can only be changed by dispatched events.

Reply:
I added this in `@remarks` for the interface